pkg/policy/validate.go

```diff
@@ -850,23 +858,27 @@ func checkClusterResourceInMatchAndExclude(rule kyverno.Rule, clusterResources [
 	if len(rule.ExcludeResources.ResourceDescription.Namespaces) > 0 {
 		return fmt.Errorf("namespaced cluster policy : field namespaces not allowed in exclude.resources")
 	}
-	// Contains "Cluster Wide Resources" in Match->ResourceDescription->Kinds
-	for _, kind := range rule.MatchResources.ResourceDescription.Kinds {
-		for _, k := range clusterResources {
-			if kind == k {
-				return fmt.Errorf("namespaced policy : cluster type value '%s' not allowed in match.resources.kinds", kind)
+
+	if !mock {
+		// Contains "Cluster Wide Resources" in Match->ResourceDescription->Kinds
+		for _, kind := range rule.MatchResources.ResourceDescription.Kinds {
+			for _, k := range clusterResources {
+				if kind == k {
+					return fmt.Errorf("namespaced policy : cluster type value '%s' not allowed in match.resources.kinds", kind)
```

Comment:
"namespaced policy : **cluster-wide resource** '%s' is not allowed in match.resources.kinds"